src/base/utils/misc.cpp

```diff
@@ -358,12 +358,20 @@ QString Utils::Misc::friendlyUnit(qint64 bytesValue, bool isSpeed)
     if (unit == SizeUnit::Byte)
         ret = QString::number(bytesValue) + " " + unitString(unit);
     else
-        ret = Utils::String::fromDouble(friendlyVal, 1) + " " + unitString(unit);
+        ret = Utils::String::fromDouble(friendlyVal, friendlyUnitPrecision(unit)) + " " + unitString(unit);
```

Comment:
@thalieht:  To answer your question, I believe @evsh is referring to this line here. 

There is a difference between the "standard" space (typed using the spacebar), and a "no-break" space (typed using ALT-255 on Windows).  

See [https://en.m.wikipedia.org/wiki/Non-breaking_space] for details. 

In short, @evsh is asking you to use a different character in the quotes than the normal space.